### PR TITLE
added new stuffs

### PR DESCRIPTION
**Pull request by andela-aidowu**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -56,5 +56,6 @@ app.use(function(err, req, res, next) {
   });
 });
 
+//changed nothing
```
Comment (andela-asogbein) [inline app.js]: COOooooOOOl, now I understand this

Comment (andela-aidowu) [inline app.js]: you should delete this line

Comment (andela-kochu) [inline app.js]: Leave it as it was

Comment (andela-eashikodi) [inline app.js]: this is a nice illustration

Comment (andela-aidowu) [inline app.js]: Ok

Comment (andela-ofamakinde) [inline app.js]: Commenting...

**Pull request merged**